js/main.js

```diff
@@ -3,6 +3,7 @@ import {renderPopup, mapCanvas} from './map.js';
 import './form-title.js';
 import './form-price.js';
 import './form-guests-rooms.js';
+import './form-time-in-out.js';
```

Comment:
```suggestion
import './form-validation.js';
```
сделай чтобы все валидации, были в одном файле